### PR TITLE
Docupdate

### PR DESCRIPTION
**Pull request by rhpvorderman**

### CODE REVIEW

File: docs/MANUAL.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Manual
+
+```bash
+# Minimal example for option: outputVariants (VCF based)
+java -jar Biopet-0.2.0.jar tool BastyGenerateFasta --inputVcf myVCF.vcf \
```
Comment (DavyCats) [inline docs/MANUAL.md]: tool will not be needed anymore, simply call jar for tool directly

File: docs/MANUAL.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+--outputName NiceTool --outputVariants myVariants.fasta
+
+# Minimal example for option: outputConsensus (BAM based)
+java -jar Biopet-0.2.0.jar tool BastyGenerateFasta --bamFile myBam.bam \
```
Comment (DavyCats) [inline docs/MANUAL.md]: tool will not be needed anymore, simply call jar for tool directly

File: docs/MANUAL.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+--outputName NiceTool --outputConsensus myConsensus.fasta
+
+# Minimal example for option: outputConsensusVariants
+java -jar Biopet-0.2.0.jar tool BastyGenerateFasta --inputVcf myVCF.vcf --bamFile myBam.bam \
```
Comment (DavyCats) [inline docs/MANUAL.md]: tool will not be needed anymore, simply call jar for tool directly

**Review by DavyCats — CHANGES_REQUESTED** [review]

**Comment by rhpvorderman** [comment]
Done

File: docs/INSTALLATION.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#  Installation
+This tool requires Java 8 to be installed on your device. Download Java 8 
+[here](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html) 
+or install via your distribution's package manager.
+
+Download the latest version of BastyGenerateFasta [here](https://github.com/biopet/bastygeneratefasta/releases/). 
+To generate the usage run:
+```bash
+java -jar <tool>-version.jar --help
```
Comment (DavyCats) [inline docs/INSTALLATION.md]: shouldn't the jar be changed here too?

Comment (rhpvorderman) [reply docs/INSTALLATION.md]: done

**Review by DavyCats — APPROVED** [review]

**Pull request merged**